otter/integration/tests/test_convergence.py

```diff
@@ -721,3 +778,42 @@ def setUp(self):
             .addCallback(clb.wait_for_state, "ACTIVE", 600)
             for clb in self.helper.clbs])
         )
+
+    @classmethod
+    def _copy_catc_4_to_12(cls, name, method):
+        """
+        To be used to copy over methods from ConvergenceSet1 using
+        :func:`duplicate_test_methods`.  Note that the methods copied
+        should all return the scaling group, so that the group's active
+        servers can be checked against the CLB.
+        """
+        tags = getattr(method, 'tags', ())
+        if not any(tag in tags for tag in
+                   ["CATC-0{0:02d}".format(i) for i in range(4, 13)]):
```

Comment:
Non-blocking nitpicky suggestion: A set might make it clearer: `if set(tags).isdisjoint(set("CATC-0{0:02d}".format(i) for i in range(4, 13)))`. Maybe not. Its fine either way :) Also would be nice to have separate function to generate CATC names based on it's range? 
```python
def catc_names(beg, end):
   return ["CATC-0{0:02d}".format(i) for i in range(beg, end)]
```
This can be used below too unless this is not going to be used anywhere :). Again, both these suggestions are non-blocking nitpicks :).